dev-libs/libsodium/libsodium-1.0.16.ebuild

```diff
@@ -16,12 +16,13 @@ IUSE="+asm minimal static-libs +urandom cpu_flags_x86_sse4_1 cpu_flags_x86_aes"
 
 PATCHES=( "${FILESDIR}"/${PN}-1.0.10-cpuflags.patch )
 
-src_prepare() {
+multilib_src_prepare() {
 	default
 	eautoreconf
 }
 
-src_configure() {
+multilib_src_configure() {
+	ECONF_SOURCE=${S}
 	local myconf
 
 	# --disable-pie needed on x86, bug #512734
```

Comment:
`use x86` won't work correctly for multilib, compare against `${ABI}` instead.

Reply:
Fixed with newest commit.